src/components/RecipeItemModal/RecipeItemModal.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import {
+  ModalContainer,
+  ModalImage,
+  CloseButton,
+  ButtonWrapper,
+  ImageWrapper,
+  InfoWrapper,
+  Ingredients,
+  AdditionalInfo,
+} from "./styles";
+import closeButtonIcon from "../../icons/closeButtonIcon.svg";
+
+const RecipeModalWindow = ({ recipeData, customOnClick }) => {
+  const totalWeight = recipeData?.totalWeight ? recipeData.totalWeight : 1;
+  const KCAL = recipeData?.nutrients.ENERC_KCAL.quantity
+    ? recipeData.nutrients.ENERC_KCAL.quantity
+    : 0;
+
+  return (
+    <ModalContainer onClick={(e) => e.stopPropagation()}>
+      <ButtonWrapper>
```

Comment:
Could you please move close button to modal window component and fix scrolling issues?

https://user-images.githubusercontent.com/14818162/206129039-21f1a468-b760-420e-8076-5c626c19ca7f.mp4